vshavel/tables.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+
+    <meta charset="UTF-8">
+
+    <title> Tables</title>
+
+</head>
+
+
+<body>
+
+<table width="600" border="1" align="center">
+
+<col width="20%"><col width="20%"><col width="20%"><col width="20%"><col width="20%">
+	
+	<caption><i> The weather in Belarus </i></caption>
+	
+	<tr>
+		<th>City</th><th>Morning</th><th>Noon</th><th>Evening</th><th>Night</th>
+	</tr>
+	
+	<tr>
+		<th>Minsk</th><td>-5&#176;</td><td>+5<sup><small>o</small></sup></td><td rowspan="2">+2&deg;</td><td>-10&#176;</td>
+	</tr>
+	
+	<tr>
+		<th>Brest</th><td>0&deg;</td><td>+9&#176;</td><td>-4&#176;</td>
+	</tr>
+	
+</table>
+
+<br>
```

Comment:
It will better to add hr

Reply:
All these heads and others lines
It’s not mine
I will try to solve my problem with git and sent new pull request to you